recipes/mocha-to-node-test-runner/tests/input/1_basic.js

```diff
@@ -0,0 +1,10 @@
+const assert = require('assert');
+
+describe('Array', function() {
+	describe.skip('#indexOf()', function() {
+		it('should return -1 when the value is not present', function() {
+			const arr = [1, 2, 3];
+			assert.strictEqual(arr.indexOf(4), -1);
+		});
```

Comment:
maybe add an other `it` that use arrow function as call back 
and a fonction as arg like
```js
function myBasicTest = () => {
  assert(true)
}
```

The second case will be can super complicated to migrate. 
in a first time let's just have test that proof that's ins't handled and document it.
In second time (follow up pr) having systems that handle this case. The logic to catch a refercen is already furnished by the codemod context https://docs.codemod.com/jssg/semantic-analysis